api/src/main/java/org/openmrs/module/shr/cdahandler/processor/document/impl/DocumentProcessorImpl.java

```diff
@@ -419,12 +412,16 @@ private void voidVisitData(Visit oldVisit, II newId) {
 				Context.getObsService().voidObs(obs, voidReason);
 				if(obs.getAccessionNumber() != null)
 				{
```

Comment:
If you notice some brackets in new line try to correct this to follow the conventions given e.g. by Oracle.[Here is a document with conventions](http://www.oracle.com/technetwork/java/codeconventions-150003.pdf). A bracket should be in the same line with _if_, _for_, _else_, class definition etc.

Remove line break, add spaces.